<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 Querying Additional Datasets | _main.knit</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.43 and GitBook 2.6.7" />

  <meta property="og:title" content="6 Querying Additional Datasets | _main.knit" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 Querying Additional Datasets | _main.knit" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="chap5.html"/>
<link rel="next" href="chap7.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> p53motifDB</a></li>
<li class="chapter" data-level="2" data-path="chap2.html"><a href="chap2.html"><i class="fa fa-check"></i><b>2</b> General Information</a>
<ul>
<li class="chapter" data-level="2.1" data-path="chap2.html"><a href="chap2.html#web-access"><i class="fa fa-check"></i><b>2.1</b> Web access</a></li>
<li class="chapter" data-level="2.2" data-path="chap2.html"><a href="chap2.html#access-to-raw-data"><i class="fa fa-check"></i><b>2.2</b> Access to raw data</a></li>
<li class="chapter" data-level="2.3" data-path="chap2.html"><a href="chap2.html#feedback"><i class="fa fa-check"></i><b>2.3</b> Feedback</a></li>
<li class="chapter" data-level="2.4" data-path="chap2.html"><a href="chap2.html#general-use"><i class="fa fa-check"></i><b>2.4</b> General Use</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="chap3.html"><a href="chap3.html"><i class="fa fa-check"></i><b>3</b> The Main Data Table</a>
<ul>
<li class="chapter" data-level="3.1" data-path="chap3.html"><a href="chap3.html#changing-the-number-of-displayed-rows."><i class="fa fa-check"></i><b>3.1</b> Changing the number of displayed rows.</a></li>
<li class="chapter" data-level="3.2" data-path="chap3.html"><a href="chap3.html#filter-table-by-search-term."><i class="fa fa-check"></i><b>3.2</b> Filter table by search term.</a></li>
<li class="chapter" data-level="3.3" data-path="chap3.html"><a href="chap3.html#column-sorting"><i class="fa fa-check"></i><b>3.3</b> Column sorting</a></li>
<li class="chapter" data-level="3.4" data-path="chap3.html"><a href="chap3.html#hyperlinks"><i class="fa fa-check"></i><b>3.4</b> Hyperlinks</a></li>
<li class="chapter" data-level="3.5" data-path="chap3.html"><a href="chap3.html#horizontal-scrolling"><i class="fa fa-check"></i><b>3.5</b> Horizontal Scrolling</a></li>
<li class="chapter" data-level="3.6" data-path="chap3.html"><a href="chap3.html#row-display-characteristics"><i class="fa fa-check"></i><b>3.6</b> Row Display Characteristics</a></li>
<li class="chapter" data-level="3.7" data-path="chap3.html"><a href="chap3.html#page-navigation-selectors"><i class="fa fa-check"></i><b>3.7</b> Page Navigation Selectors</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="chap4.html"><a href="chap4.html"><i class="fa fa-check"></i><b>4</b> Filtering</a>
<ul>
<li class="chapter" data-level="4.1" data-path="chap4.html"><a href="chap4.html#basic-filters"><i class="fa fa-check"></i><b>4.1</b> Basic Filters</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="chap4.html"><a href="chap4.html#gene-symbolhgnc-filtering"><i class="fa fa-check"></i><b>4.1.1</b> Gene Symbol/HGNC Filtering</a></li>
<li class="chapter" data-level="4.1.2" data-path="chap4.html"><a href="chap4.html#genomic-coordinate-filtering-by-chromosome-and-location"><i class="fa fa-check"></i><b>4.1.2</b> Genomic coordinate filtering by chromosome and location</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="chap4.html"><a href="chap4.html#p53-binding-filters"><i class="fa fa-check"></i><b>4.2</b> p53 Binding Filters</a></li>
<li class="chapter" data-level="4.3" data-path="chap4.html"><a href="chap4.html#chromatin-filters"><i class="fa fa-check"></i><b>4.3</b> Chromatin Filters</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="chap4.html"><a href="chap4.html#candidate-cis-regulatory-elements-ccre"><i class="fa fa-check"></i><b>4.3.1</b> Candidate cis-regulatory elements (cCRE)</a></li>
<li class="chapter" data-level="4.3.2" data-path="chap4.html"><a href="chap4.html#dnase-hypersensitive-sites-dhs"><i class="fa fa-check"></i><b>4.3.2</b> DNAse Hypersensitive Sites (DHS)</a></li>
<li class="chapter" data-level="4.3.3" data-path="chap4.html"><a href="chap4.html#chromhmm"><i class="fa fa-check"></i><b>4.3.3</b> chromHMM</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="chap4.html"><a href="chap4.html#d-genome-information"><i class="fa fa-check"></i><b>4.4</b> 3D Genome Information</a></li>
<li class="chapter" data-level="4.5" data-path="chap4.html"><a href="chap4.html#genome-variation"><i class="fa fa-check"></i><b>4.5</b> Genome Variation</a></li>
<li class="chapter" data-level="4.6" data-path="chap4.html"><a href="chap4.html#other-genomic-variables"><i class="fa fa-check"></i><b>4.6</b> Other Genomic Variables</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="chap4.html"><a href="chap4.html#intergenic-or-intragenic"><i class="fa fa-check"></i><b>4.6.1</b> Intergenic or Intragenic?</a></li>
<li class="chapter" data-level="4.6.2" data-path="chap4.html"><a href="chap4.html#repetitive-elements"><i class="fa fa-check"></i><b>4.6.2</b> Repetitive Elements</a></li>
<li class="chapter" data-level="4.6.3" data-path="chap4.html"><a href="chap4.html#encode-blacklist"><i class="fa fa-check"></i><b>4.6.3</b> ENCODE BlackList</a></li>
<li class="chapter" data-level="4.6.4" data-path="chap4.html"><a href="chap4.html#potential-cg-methylation-sites"><i class="fa fa-check"></i><b>4.6.4</b> Potential CG Methylation Sites</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="chap5.html"><a href="chap5.html"><i class="fa fa-check"></i><b>5</b> Exporting Filtered Data</a>
<ul>
<li class="chapter" data-level="5.1" data-path="chap5.html"><a href="chap5.html#select-columns-to-export-button"><i class="fa fa-check"></i><b>5.1</b> Select Columns to Export Button</a></li>
<li class="chapter" data-level="5.2" data-path="chap5.html"><a href="chap5.html#download-filtered-data-button"><i class="fa fa-check"></i><b>5.2</b> Download Filtered Data Button</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="chap6.html"><a href="chap6.html"><i class="fa fa-check"></i><b>6</b> Querying Additional Datasets</a>
<ul>
<li class="chapter" data-level="6.1" data-path="chap6.html"><a href="chap6.html#clinvar"><i class="fa fa-check"></i><b>6.1</b> ClinVar</a></li>
<li class="chapter" data-level="6.2" data-path="chap6.html"><a href="chap6.html#dbsnp"><i class="fa fa-check"></i><b>6.2</b> dbSNP</a></li>
<li class="chapter" data-level="6.3" data-path="chap6.html"><a href="chap6.html#encode-dhs"><i class="fa fa-check"></i><b>6.3</b> ENCODE DHS</a></li>
<li class="chapter" data-level="6.4" data-path="chap6.html"><a href="chap6.html#repeatmasker"><i class="fa fa-check"></i><b>6.4</b> RepeatMasker</a></li>
<li class="chapter" data-level="6.5" data-path="chap6.html"><a href="chap6.html#riege-p53-meta"><i class="fa fa-check"></i><b>6.5</b> Riege p53 Meta</a></li>
<li class="chapter" data-level="6.6" data-path="chap6.html"><a href="chap6.html#remap-2022"><i class="fa fa-check"></i><b>6.6</b> ReMap 2022</a></li>
<li class="chapter" data-level="6.7" data-path="chap6.html"><a href="chap6.html#abc-3d-genome"><i class="fa fa-check"></i><b>6.7</b> ABC 3D Genome</a></li>
<li class="chapter" data-level="6.8" data-path="chap6.html"><a href="chap6.html#genehancer"><i class="fa fa-check"></i><b>6.8</b> GeneHancer</a></li>
<li class="chapter" data-level="6.9" data-path="chap6.html"><a href="chap6.html#mcf10a-microc"><i class="fa fa-check"></i><b>6.9</b> MCF10A MicroC</a></li>
<li class="chapter" data-level="6.10" data-path="chap6.html"><a href="chap6.html#hct116-pchic"><i class="fa fa-check"></i><b>6.10</b> HCT116 pcHIC</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="chap7.html"><a href="chap7.html"><i class="fa fa-check"></i><b>7</b> Database Summary Statistics</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chap6" class="section level1 hasAnchor" number="6">
<h1><span class="header-section-number">6</span> Querying Additional Datasets<a href="chap6.html#chap6" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Once users have completed all of their filtering using the Basic Filters on the left-hand side of the p53motifDB Shiny App (covered in Chapter <a href="chap3.html#chap3">3</a>), they are left with a main data table containing the p53RE that have the characteristics of interest. Chapter <a href="chap5.html#chap5">5</a> describes how to export the Main Data Table information, and this chapter describes how users can download a series of “accessory” datasets that provide additional information about the p53RE in that filtered Main Data Table. Each button allows users to export new information about their p53RE. This works much like cross-referencing in sql or other relational database schemes. Once the Main Data Table filters are applied, users can export accessory data using the buttons at the bottom of the App (shown in Figure <a href="chap3.html#fig:nav">3.1</a>). Data are exported as comma separated value (.csv) files via a system dialog box where files can be named and saved to the location of the users choice.</p>
<p>Data are exported for ALL p53RE found in the filtered Main Data Table, even if there are no records for the particular accessory data. If there are no records, the p53RE unique_id (chromosome number, start location, and stop location) is displayed, but other columns will display “NA”.</p>
<p>We recommend applying multiple Basic Filters before exporting accessory data. Users interested in all information contained in the accessory data tables can find these on our <a href="https://doi.org/10.5281/zenodo.13351804">Zenodo repository</a>, which also contains a pre-compiled sqlite database for use of advanced query language.</p>
<p>In the next subsections, we will go through the types of data that users can export from these accessory tables.</p>
<div id="clinvar" class="section level2 hasAnchor" number="6.1">
<h2><span class="header-section-number">6.1</span> ClinVar<a href="chap6.html#clinvar" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">column_name</th>
<th align="left">information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">unique_id</td>
<td align="left">chromosome_start_stop for p53RE</td>
</tr>
<tr class="even">
<td align="left">clinVar_chrom</td>
<td align="left">chromosome where the clinVar variant is located</td>
</tr>
<tr class="odd">
<td align="left">clinVar_start</td>
<td align="left">start of chromosome position of the clinVar variant</td>
</tr>
<tr class="even">
<td align="left">clinVar_stop</td>
<td align="left">stop/end of chromosome position of the clinVar variant</td>
</tr>
<tr class="odd">
<td align="left">clinVar_id</td>
<td align="left">clinVar database standard ID</td>
</tr>
<tr class="even">
<td align="left">clinVar_ref</td>
<td align="left">the reference allele</td>
</tr>
<tr class="odd">
<td align="left">clinVar_alt</td>
<td align="left">the alternate allele</td>
</tr>
<tr class="even">
<td align="left">clinVar_type</td>
<td align="left">the type of clinVar variant; options include INSERTION, DELETION, SNV, MNV</td>
</tr>
</tbody>
</table>
</div>
<div id="dbsnp" class="section level2 hasAnchor" number="6.2">
<h2><span class="header-section-number">6.2</span> dbSNP<a href="chap6.html#dbsnp" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Single p53RE locations may contain multiple genomic variants in the dbSNP accessory dataset, so the same p53RE unique_id (location) can be found in multiple rows of the exported data. Each row will contain unique data reflecting the specific genomic variant, dbSNP ID, and other information found in additional columns.</p>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">column_name</th>
<th align="left">information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">unique_id</td>
<td align="left">chromosome_start_stop for p53RE</td>
</tr>
<tr class="even">
<td align="left">dbSNP_chrom</td>
<td align="left">chromosome where the dbSNP variant is located</td>
</tr>
<tr class="odd">
<td align="left">dbSNP_start</td>
<td align="left">start of chromosome position of the dbSNP variant</td>
</tr>
<tr class="even">
<td align="left">dbSNP_stop</td>
<td align="left">stop/end of chromosome position of the dbSNP variant</td>
</tr>
<tr class="odd">
<td align="left">rs_id</td>
<td align="left">dbSNP standard ID</td>
</tr>
<tr class="even">
<td align="left">dbSNP_ref</td>
<td align="left">the reference allele</td>
</tr>
<tr class="odd">
<td align="left">dbSNP_alt</td>
<td align="left">the alternate allele</td>
</tr>
<tr class="even">
<td align="left">dbSNP_type</td>
<td align="left">the type of dbSNP variant; options include INSERTION, DELETION, SNV, MNV</td>
</tr>
</tbody>
</table>
</div>
<div id="encode-dhs" class="section level2 hasAnchor" number="6.3">
<h2><span class="header-section-number">6.3</span> ENCODE DHS<a href="chap6.html#encode-dhs" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>DNase Hypersensitive Sites (DHS) have been surveyed across hundred of cell types and conditions. Therefore, the same p53RE unique_id (location) may be found in multiple rows of the exported data, reflecting the observed cell types where that genomic location is accessible to DNase.</p>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">column_name</th>
<th align="left">information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">unique_id</td>
<td align="left">chromosome_start_stop for p53RE</td>
</tr>
<tr class="even">
<td align="left">dhs_loc</td>
<td align="left">chromosome_start_stop for the DHS (Dnase Hypersensite Site)</td>
</tr>
<tr class="odd">
<td align="left">dhs_intersection</td>
<td align="left">Is the p53RE found in a DHS region (YES/NO)</td>
</tr>
<tr class="even">
<td align="left">dhs_cell_id</td>
<td align="left">the ENCODE cell line designation/number</td>
</tr>
<tr class="odd">
<td align="left">dhs_score</td>
<td align="left">the score of the DHS, ranges from 1-1000. The larger the number, the larger the DHS signal.</td>
</tr>
<tr class="even">
<td align="left">dhs_cell_type</td>
<td align="left">the cell type where the DHS was identified; linked to the dhs_cell_id</td>
</tr>
</tbody>
</table>
</div>
<div id="repeatmasker" class="section level2 hasAnchor" number="6.4">
<h2><span class="header-section-number">6.4</span> RepeatMasker<a href="chap6.html#repeatmasker" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>There are many different types of repetitive elements, ranging from simple DNA repeats to virus-derived elements. This accessory dataset provide additional information about the full location fo the repetitive element, as well as the repeat element name, family, and class.</p>
<table>
<thead>
<tr class="header">
<th align="left">column_name</th>
<th align="left">information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">unique_id</td>
<td align="left">chromosome_start_stop for p53RE</td>
</tr>
<tr class="even">
<td align="left">rmsk_chrom</td>
<td align="left">chromosome where the repetitive element is located</td>
</tr>
<tr class="odd">
<td align="left">rmsk_start</td>
<td align="left">start of chromosome position of the repetitive element</td>
</tr>
<tr class="even">
<td align="left">rmsk_stop</td>
<td align="left">stop/end of chromosome position of the repetitive element</td>
</tr>
<tr class="odd">
<td align="left">repeat_name</td>
<td align="left">the standardized name of the repeat</td>
</tr>
<tr class="even">
<td align="left">repeat_class</td>
<td align="left">the class of the repeat element</td>
</tr>
<tr class="odd">
<td align="left">repeat_family</td>
<td align="left">the family of the repeat element</td>
</tr>
<tr class="even">
<td align="left">rmsk_intersection</td>
<td align="left">is the p53RE found in a repetitive element (YES/NO)</td>
</tr>
</tbody>
</table>
</div>
<div id="riege-p53-meta" class="section level2 hasAnchor" number="6.5">
<h2><span class="header-section-number">6.5</span> Riege p53 Meta<a href="chap6.html#riege-p53-meta" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">column_name</th>
<th align="left">information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">unique_id</td>
<td align="left">chromosome_start_stop for p53RE</td>
</tr>
<tr class="even">
<td align="left">fischer_chrom</td>
<td align="left">chromosome where the full p53 ChIP-seq peak is located</td>
</tr>
<tr class="odd">
<td align="left">fischer_start</td>
<td align="left">start of chromosome position of the full p53 ChIP-seq peak</td>
</tr>
<tr class="even">
<td align="left">fischer_stop</td>
<td align="left">stop/end of chromosome position of the full p53 ChIP-seq peak</td>
</tr>
<tr class="odd">
<td align="left">fischer_loc</td>
<td align="left">the full location of the p53 ChIP-seq peak in chrom_start_stop format</td>
</tr>
<tr class="even">
<td align="left">fischer_obs</td>
<td align="left">The number of datasets from the Riege et al. meta-analysis where this location is a called p53 ChIP-seq peak/binding site (range of 5-28)</td>
</tr>
<tr class="odd">
<td align="left">fischer_percentages</td>
<td align="left">The percent of datasets where this location is a p53 binding site/peak.</td>
</tr>
</tbody>
</table>
</div>
<div id="remap-2022" class="section level2 hasAnchor" number="6.6">
<h2><span class="header-section-number">6.6</span> ReMap 2022<a href="chap6.html#remap-2022" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">column_name</th>
<th align="left">information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">unique_id</td>
<td align="left">chromosome_start_stop for p53RE</td>
</tr>
<tr class="even">
<td align="left">remap_loc</td>
<td align="left">the full location of the p53 ChIP-seq peak in chrom_start_stop format</td>
</tr>
<tr class="odd">
<td align="left">remap_chrom</td>
<td align="left">chromosome where the full p53 ChIP-seq peak is located</td>
</tr>
<tr class="even">
<td align="left">remap_start</td>
<td align="left">start of chromosome position of the full p53 ChIP-seq peak</td>
</tr>
<tr class="odd">
<td align="left">remap_stop</td>
<td align="left">stop/end of chromosome position of the full p53 ChIP-seq peak</td>
</tr>
<tr class="even">
<td align="left">remap_cell</td>
<td align="left">The type of cell where the p53 ChIP experiment was performed</td>
</tr>
<tr class="odd">
<td align="left">remap_obs</td>
<td align="left">The number of ReMap datasets where this location was a p53 peak/binding site</td>
</tr>
<tr class="even">
<td align="left">remap_percentage</td>
<td align="left">The percent of datasets where this location was a p53 peak/binding site</td>
</tr>
</tbody>
</table>
</div>
<div id="abc-3d-genome" class="section level2 hasAnchor" number="6.7">
<h2><span class="header-section-number">6.7</span> ABC 3D Genome<a href="chap6.html#abc-3d-genome" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The Activity by Contact (ABC) dataset aims to “connect” p53RE to potential regulated genes. The original manuscript describes this work more in detail, but users of this database can obtain relevant information about potential direct p53 target genes, including the gene connected to the p53RE, the type of cell where this interaction takes place, and the location of the p53RE relative to the gene.</p>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">column_name</th>
<th align="left">information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">unique_id</td>
<td align="left">chromosome_start_stop for p53RE</td>
</tr>
<tr class="even">
<td align="left">abc_chr</td>
<td align="left">chromosome where the gene connected to the p53RE is located</td>
</tr>
<tr class="odd">
<td align="left">abc_TargetGeneTSS_start</td>
<td align="left">start position for the TSS of the gene</td>
</tr>
<tr class="even">
<td align="left">abc_TargetGeneTSS_end</td>
<td align="left">stop position for the TSS of the gene</td>
</tr>
<tr class="odd">
<td align="left">abc_TargetGene_loc</td>
<td align="left">TSS location for the p53RE-connected gene in chrom_start_stop format</td>
</tr>
<tr class="even">
<td align="left">TargetGene</td>
<td align="left">name of the Target Gene in HGNC/gene symbol format</td>
</tr>
<tr class="odd">
<td align="left">class</td>
<td align="left">the type of regulatory element where the p53RE is located (promoter, intergenic, genic)</td>
</tr>
<tr class="even">
<td align="left">TargetGeneIsExpressed</td>
<td align="left">Based on RNA-seq or other evidence, is the target gene expressed in the cell type of interest</td>
</tr>
<tr class="odd">
<td align="left">isSelfPromoter</td>
<td align="left">Is the location of the p53RE within a promoter, and does that promoter likely regulate the gene</td>
</tr>
<tr class="even">
<td align="left">ABC.Score</td>
<td align="left">a proprietary score from the ABC model reflecting the confidence that the p53RE-containing element regulates the listed gene.</td>
</tr>
<tr class="odd">
<td align="left">CellType</td>
<td align="left">the type of cell where the prediction for regulation of the promoter by the p53RE-containing element was made</td>
</tr>
<tr class="even">
<td align="left">ABC_intersection</td>
<td align="left">Is this p53RE found in a region covered by the ABC predictions</td>
</tr>
</tbody>
</table>
</div>
<div id="genehancer" class="section level2 hasAnchor" number="6.8">
<h2><span class="header-section-number">6.8</span> GeneHancer<a href="chap6.html#genehancer" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Like the ABC dataset, the GeneHancer dataset aims to “connect” p53RE to potential regulated genes. The original manuscript describes this work more in detail, but the current release is proprietary and owned by the academic institution and non-profit that runs the <a href="https://www.genecards.org/Guide/Datasets">GeneCards website</a>. The full dataset for ALL GeneHancer data can be obtained from this company for academic and non-profit users by request.</p>
<p>In our database, users can obtain relevant information about potential direct p53 target genes determined by the GeneHancer group, including the gene connected to the p53RE, the “strength” of the association, and the predicted type of regulatory element where the p53RE is located.</p>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">column_name</th>
<th align="left">information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">unique_id</td>
<td align="left">chromosome_start_stop for p53RE</td>
</tr>
<tr class="even">
<td align="left">gh_loc</td>
<td align="left">the location of the p53RE-containing regulatory element connected to the gene of interest.</td>
</tr>
<tr class="odd">
<td align="left">GHid</td>
<td align="left">a unique identifier for the GeneHancer database</td>
</tr>
<tr class="even">
<td align="left">gh_connected_gene</td>
<td align="left">name of the Target Gene in HGNC/gene symbol format</td>
</tr>
<tr class="odd">
<td align="left">gh_score</td>
<td align="left">a proprietary score from the GeneHancer model reflecting the confidence that the p53RE-containing element regulates the listed gene.</td>
</tr>
<tr class="even">
<td align="left">gh_elite</td>
<td align="left">Elite sites represent the highest confidence that the gene is regulated by the p53RE-containing element(0=no, 1=yes)</td>
</tr>
<tr class="odd">
<td align="left">gh_regulatory_element_type</td>
<td align="left">the type of regulatory element where the p53RE is found, as defined by GeneHancer</td>
</tr>
</tbody>
</table>
</div>
<div id="mcf10a-microc" class="section level2 hasAnchor" number="6.9">
<h2><span class="header-section-number">6.9</span> MCF10A MicroC<a href="chap6.html#mcf10a-microc" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">column_name</th>
<th align="left">information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">unique_id</td>
<td align="left">chromosome_start_stop for p53RE</td>
</tr>
<tr class="even">
<td align="left">bait_chrom</td>
<td align="left">chromosome location for the bait region</td>
</tr>
<tr class="odd">
<td align="left">bait_start</td>
<td align="left">chromosome start position for the bait</td>
</tr>
<tr class="even">
<td align="left">bait_end</td>
<td align="left">chromosome stop position for the bait</td>
</tr>
<tr class="odd">
<td align="left">prey_chrom</td>
<td align="left">chromosome location for the prey region</td>
</tr>
<tr class="even">
<td align="left">prey_start</td>
<td align="left">chromosome start position for the prey</td>
</tr>
<tr class="odd">
<td align="left">prey_end</td>
<td align="left">chromosome stop position for the prey</td>
</tr>
<tr class="even">
<td align="left">bait_gene_name</td>
<td align="left">the HGNC/gene symbol for the gene of interest</td>
</tr>
<tr class="odd">
<td align="left">baitprey_interaction</td>
<td align="left">is the p53RE found within the bait region or the prey region?</td>
</tr>
</tbody>
</table>
</div>
<div id="hct116-pchic" class="section level2 hasAnchor" number="6.10">
<h2><span class="header-section-number">6.10</span> HCT116 pcHIC<a href="chap6.html#hct116-pchic" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">column_name</th>
<th align="left">information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">unique_id</td>
<td align="left">chromosome_start_stop for p53RE</td>
</tr>
<tr class="even">
<td align="left">bait_chrom</td>
<td align="left">chromosome location for the bait region</td>
</tr>
<tr class="odd">
<td align="left">bait_start</td>
<td align="left">chromosome start position for the bait</td>
</tr>
<tr class="even">
<td align="left">bait_end</td>
<td align="left">chromosome stop position for the bait</td>
</tr>
<tr class="odd">
<td align="left">prey_chrom</td>
<td align="left">chromosome location for the prey region</td>
</tr>
<tr class="even">
<td align="left">prey_start</td>
<td align="left">chromosome start position for the prey</td>
</tr>
<tr class="odd">
<td align="left">prey_end</td>
<td align="left">chromosome stop position for the prey</td>
</tr>
<tr class="even">
<td align="left">baitprey_interaction</td>
<td align="left">is the p53RE found within the bait region or the prey region?</td>
</tr>
<tr class="odd">
<td align="left">bait_ensembl_gene_id</td>
<td align="left">the ENSEMBL id for the gene of interest</td>
</tr>
<tr class="even">
<td align="left">bait_hgnc</td>
<td align="left">the HGNC/gene symbol for the gene of interest</td>
</tr>
</tbody>
</table>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="chap5.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="chap7.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": null,
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
